libs/wingsdk/src/platform/parameter-registrar.ts

```diff
@@ -0,0 +1,76 @@
+import { Construct } from "constructs";
+import { PlatformParameter } from "./platform-parameter";
+import { loadPlatformSpecificValues } from "./util";
+
+/**
+ * Parameter Registrar
+ *
+ * This construct is used to register and validate platform parameters.
+ */
+export class ParameterRegistrar extends Construct {
```

Comment:
🤦 damn! I didnt even think about using JSON Schema to model these platform parameters. 

Honestly, I dont think it would be too difficult and I should be able to model all the same features such as `choices`, and `dependencies` using it, Ill play with it this morning.